examples/with-typescript-pages/components/CounterCard.tsx

```diff
@@ -0,0 +1,29 @@
+import Vue from 'vue';
+
+export default Vue.extend({
+  name: 'CounterCard',
+  data() {
+    return {
+      password: ''
+    };
+  },
+  render() {
+    return (
+      <div>
+        <div>Count: {this.$store.state.count}</div>
+        <button onClick={e => this.$store.commit('increment')}>+</button>
+        <button onClick={e => this.$store.commit('decrement')}>-</button>
+        <input
+          value={this.password}
+          onInput={e => (this.password = e.target.value)}
+        />
+        <nuxt-link
```

Comment:
I suggest using  `<NuxtLink`

Reply:
Where can I import `NuxtLink` component from? 
Typescript does type-check differently between `NuxtLink` & `nuxt-link`, you can check it here https://www.typescriptlang.org/docs/handbook/jsx.html